formula/phplint.rb

```diff
@@ -0,0 +1,63 @@
+class Phplint < Formula
+  desc "Validator and documentator for PHP 5 and 7 programs"
+  homepage "http://www.icosaedro.it/phplint/"
+  url "http://www.icosaedro.it/phplint/phplint-3.0_20160307.tar.gz"
+  version "3.0-20160307"
+  sha256 "7a361166d1a6de707e6728828a6002a6de69be886501853344601ab1da922e7b"
+
+  bottle :unneeded
+
+  depends_on "php@7.1"
+
+  def install
+    inreplace "php", "/opt/php/bin/php", "#{Formula["php@7.1"].opt_bin}/php"
+    inreplace "phplint.tcl", "\"MISSING_PHP_CLI_EXECUTABLE\"", "#{opt_libexec}/php"
+    inreplace "phplint.tcl", "set opts\(phplint_dir\) \[pwd\]", "set opts(phplint_dir) #{libexec}"
```

Comment:
This is not update safe. I'll see if I can find a more sustainable solutoin since the path is stored in a file in the HOME directory.

Reply:
The program throws an error
```
FATAL ERROR in /usr/local/Cellar/phplint/3.0-20160307/libexec/stdlib/autoload.php: unexpected php.ini loaded: /usr/local/opt/phplint/libexec/stdlib/php.ini, expected: /usr/local/Cellar/phplint/3.0-20160307/libexec/stdlib/php.ini
```
The traces back to
```
/*. private void .*/ function check_php_ini()
{
	$expected = __DIR__ . DIRECTORY_SEPARATOR . "php.ini";
	$actual = php_ini_loaded_file();
	if( $actual === FALSE )
		die("FATAL ERROR in " . __FILE__ . ": no php.ini file loaded; expected $expected");
	if( $actual !== $expected )
		die("FATAL ERROR in " . __FILE__ . ": unexpected php.ini loaded: $actual, expected: $expected");
}
```

Reply:
Since we can't really test the gui and tcl should we just keep it as an "on your own risk" or remove it (or not link it to bin) ? Otherwise we need to patch the function most likely.

Reply:
I'd just remove it for now.